osquery/tables/system/darwin/authdb.mm

```diff
@@ -0,0 +1,99 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <osquery/core.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+
+#include "osquery/tables/system/darwin/authdb.h"
+
+namespace osquery {
+namespace tables {
+
+QueryData genOSXAuthDB(QueryContext& context) {
```

Comment:
Most likely, have not run it through tests...
First stab :)
Will take a look on Monday. Thanks for the review, it's helpful.